modules/eventing/pages/eventing-rbac.adoc

```diff
@@ -29,7 +29,18 @@ A bucket.scope combination is used for identifying functions belonging to the sa
 Only the "Eventing Full Admin" role and also the "Full Admin" role can set the bucket.scope to  *+*+.+*+*; all other Eventing non-privileged users need to define a *Function Scope* for their Eventing functions that references an existing resource of bucket.scope. 
 This provides role based isolation of Eventing functions between non-privileged users
 
-Typically you should set Function Scope to the bucket.scope that holds the collection that is the source of your mutations to your Eventing Function.  This best practice ensures that you _do not_  inadvertently cause an Eventing Function to undeploy by removing a *Function Scope* pointing to a resource that is not required for the function to run.
+// tag::rbac-change-warning[]
```

Comment:
Does this work? I thought the file needed to be set `:page-partial:` to get picked up from the change in the other PR?

Reply:
I think it should, but I may be wrong. Just checking again.

Reply:
Okay, got it working in the preview, by inserting an underscore in front of the @cloud … well, I say 'I', but IntelliJ corrected it by adding the underscore, so I'm going to have to ask the Ziulip folk what the underscore does (at a guess, it's for components without a version id).

Thinking about it though; it's probably better to just put the note in a partial, as you suggested. I'll fix it on Monday

Reply:
Yup. Dan answered my question: the underscore denotes a component without a version id (the Capella component doesn't have one). 

I'm going to leave it as it is, if that's okay. 👍🏾

Reply:
One other small point: the reason I've put the tagged section in the Capella branch is because it is unversioned, so I figure it's best to put it somewhere that won't have multiple versions.